webapp/src/Command/LoadDefaultDataCommand.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace App\Command;
+
+use Exception;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class LoadDefaultDataCommand extends Command
+{
+    /**
+     * @inheritDoc
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('domjudge:load-default-data')
+            ->setDescription('Load the default data');
```

Comment:
The description just repeats the name. Seems like a great place to better specify what "default" data is (versus the other command that loads "example" data)